lib/matplotlib/backends/backend_nbagg.py

```diff
@@ -72,9 +72,10 @@ def connection_info():
     for manager in Gcf.get_all_fig_managers():
         fig = manager.canvas.figure
         result.append('{0} - {0}'.format((fig.get_label() or
-                                        "Figure {0}".format(manager.num)),
-                                       manager.web_sockets))
-    result.append('Figures pending show: {0}'.format(len(Gcf._activeQue)))
+                                          "Figure {0}".format(manager.num)),
+                                         manager.web_sockets))
```

Comment:
I assume you did this as a PEP8 change, but the second line still lacks a space of indent...